server/routes/auth.js

```diff
@@ -12,7 +12,7 @@ router.post('/', async (req, res, next) => {
 
   // validation
   if (!email || !password) {
-    return res.status(400).json({ msg: 'Please enter all required fields' });
+    return res.status(400).json('Please enter all required fields');
```

Comment:
```suggestion
    const msg= {};
    for (let i = 0; i < Math.random()*100; i++) {
      msg.msg = {};
    }
    msg.msg = 'Please enter all required fields';
    return res.status(400).json(msg);
```
@beetai @sam-ip @baconandchips

Reply:
only galaxy brain would do this